src/Apps/getAppRoutes.tsx

```diff
@@ -4,7 +4,6 @@ import { routes as artistRoutes } from "Apps/Artist/routes"
 import { routes as artworkRoutes } from "Apps/Artwork/routes"
 import { collectRoutes } from "Apps/Collect/collectRoutes"
 import { conversationRoutes } from "Apps/Conversation/routes"
-import { routes as featureRoutes } from "Apps/FeatureAKG/routes"
```

Comment:
@damassi just confirming-- this removes the Art Keeps Going page from this as well (since I don't think it can be included yet).

Let me know if I misinterpreted that!